tools/sense_studio/video_recording.py

```diff
@@ -14,6 +15,12 @@
 video_recording_bp = Blueprint('video_recording_bp', __name__)
 
 
+@video_recording_bp.route('/ffmpeg-check', methods=['GET'])
+def check_ffmpeg():
+    ffmpeg_installed = importlib.util.find_spec('ffmpeg')
```

Comment:
I think what this checks is the availability of the python module `ffmpeg`. What we need to check however is if `ffmpeg` itself is installed system-wide.